src/main/java/at/hannibal2/skyhanni/features/garden/farming/FarmingArmorDrops.kt

```diff
@@ -88,9 +93,50 @@ class FarmingArmorDrops {
     }
 
     private fun checkArmor() {
-        val armorPieces = InventoryUtils.getArmor()
+        val armorPieces = InventoryUtils.getArmor(1500)
             .mapNotNull { it?.getInternalName_old() }
             .count { armorPattern.matcher(it).matches() }
         hasArmor = armorPieces > 1
     }
+
+    @SubscribeEvent
+    fun onRepoReload(event: RepositoryReloadEvent) {
+        try {
+            armorDropJson = event.getConstant("ArmorDrops") ?: error("ArmorDrops not found in repo")
+
+        } catch (e: Exception) {
+            e.printStackTrace()
+            LorenzUtils.error("error in RepositoryReloadEvent")
+        }
```

Comment:
Please do not save the JSON file, but instead immediately parse it into a narrow value space. (See https://lexi-lambda.github.io/blog/2019/11/05/parse-don-t-validate/)